ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent-R2-CUI-nonprod.json

```diff
@@ -627,6 +627,18 @@
       }
     ]
   },
+  {
+    "CaseTypeID": "CIVIL",
+    "CaseEventID": "INVALID_HWF_REFERENCE",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "caseworker-civil-admin"
```

Comment:
As we discussed yesterday,we only need "cui-admin-profile" here.

Reply:
done